test/types/schema.test.ts

```diff
@@ -1007,7 +1014,7 @@ function gh12869() {
 
   const dbExample = new Schema(
     {
-      active: { type: String, enum: ['foo', 'bar'], required: true }
+      active: { type: String, enum: ['foo', 'bar'] as ['foo', 'bar'], required: true }
```

Comment:
is this casting really required instead of just `as const`?

Reply:
This particular one isn't required, I'm changing.